utils/benchmark_config.py

```diff
@@ -234,7 +234,7 @@ def _parse_unit(cls, unit):
       raise ValueError('\'%s\' does not contain a Blaze command (e.g. build)' %
                        unit['command'])
     options = []
-    while full_command_tokens and full_command_tokens[0].startswith('--'):
+    while full_command_tokens and full_command_tokens[0].startswith('--') and full_command_tokens[0] != '--':
```

Comment:
I had to read this three times to see the difference.

Consider adding a comment with an example command line and explaining what part is parsed here.